.vscode/launch.json

```diff
@@ -38,7 +38,13 @@
       "${workspaceFolder}/packages/gimbal/src/index.ts",
       "audit",
       "--cwd",
-      "${workspaceFolder}/packages/${input:appInput}"
+      "${workspaceFolder}/packages/${input:appInput}",
+      "--no-heap-snapshot",
+      "--no-calculate-unused-source",
+      "--route",
+      "/",
+      "--route",
+      "/"
```

Comment:
this stuff only temp while testing, will need to handle multiple different routes tho...

Reply:
Also need to work with more than 2